Gemfile.lock

```diff
@@ -327,7 +327,7 @@ DEPENDENCIES
   simplecov
 
 RUBY VERSION
-   ruby 3.2.6p234
+   ruby 3.2.7p253
 
 BUNDLED WITH
-   2.2.33
+   2.4.19
```

Comment:
Getting bundler to also use a newer bundler version was harder than I thought, since bunlder downgrades itself for compatibility:
```
Bundler 2.4.19 is running, but your lockfile was generated with 2.2.33. Installing Bundler 2.2.33 and restarting using that version.
```

In the end I removed the `Gemfile.lock` ran bundle install and just applied the changes to the lockfile I wanted.